js/ui/deskletManager.js

```diff
@@ -77,7 +77,7 @@ function init(){
         global.settings.connect('changed::' + DESKLET_SNAP_INTERVAL_KEY, _onDeskletSnapChanged);
 
         deskletsLoaded = true;
-        enableMouseTracking(true);
+        updateMouseTracking();
```

Comment:
This could be moved to line 73 after `definitions = getDefinitions()`. The array won't change after that point until a desklet is added or removed.

Reply:
The mouse tracker source calls methods on the desklet instances, so I figured after the promise was the proper place to put it to ensure the function could only ever be called after the first instance was fully loaded. Granted it would have to take > 500ms for the instance to load for it to cause an issue, is my logic incorrect here? I don't really know the internals of the extensions system.

OTOH if definitions[i].desklet is falsy before it fully loads, calling it immediately should be ok.

Reply:
I didn't see that it was checking the desklet itself, that makes sense to me.